ibis/expr/rules.py

```diff
@@ -622,6 +622,14 @@ def string(**arg_kwds):
     return ValueTyped(dt.string, 'not string', **arg_kwds)
 
 
+def array_array(value_type):
```

Comment:
This name is too goofy. I plan to rename what's current called an `array` to `column`.